gdx-video/src/com/badlogic/gdx/video/scenes/scene2d/VideoActor.java

```diff
@@ -0,0 +1,60 @@
+/*******************************************************************************
+ * Copyright 2023 See AUTHORS file.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ ******************************************************************************/
+
+package com.badlogic.gdx.video.scenes.scene2d;
+
+import com.badlogic.gdx.graphics.Texture;
+import com.badlogic.gdx.graphics.g2d.Batch;
+import com.badlogic.gdx.scenes.scene2d.Actor;
+import com.badlogic.gdx.utils.GdxRuntimeException;
+import com.badlogic.gdx.video.VideoPlayer;
+
+/** A simple actor that allows you to integrate a video in a 2D scene.
+ * <p>
+ * The actor takes care of updating and displaying the video on each frame. The creation, video loading, playback control, and
+ * disposing of the {@link VideoPlayer} is left to the user.
+ * <p>
+ * The actor must be removed before disposing the VideoPlayer and must not be used again after. */
+public class VideoActor extends Actor {
+	/** Creates a new video actor that displays the video from the provided player.
+	 *
+	 * @param player The video player that this actor uses. Must not be null! */
+	public VideoActor (VideoPlayer player) {
+		if (player == null) throw new GdxRuntimeException("VideoActor: player must not be null!");
+		this.player = player;
+	}
+
+	private final VideoPlayer player;
+
+	@Override
+	public void act (float delta) {
+		super.act(delta);
+		player.update();
+	}
+
+	@Override
+	public void draw (Batch batch, float parentAlpha) {
+		Texture texture = player.getTexture();
+		if (texture == null) return;
+		texture.setFilter(Texture.TextureFilter.Linear, Texture.TextureFilter.Linear);
```

Comment:
Is there a reason to set a `TextureFilter`? Is it needed to be set on every draw?

Reply:
I felt the steps were too visible from the default `Nearest` filter, when resizing the window from the examples. But to be honest, what looks best is subjective.

The filter stays on the texture once it is set, so there is no need to set it multiple times. However, setting it just once doesn't always work, as the texture object may change in some cases. So this implementation is simple and reliable rather than optimal.

Looking at it a second time however, I don't like how this doesn't allow users of VideoActor to set another filter if it fits their game better.

I could make the filter configurable in VideoActor and make the implementation more optimal. Or just remove it from VideoActor for now and let the user handle all of it. Which would you prefer for now?

Reply:
I think removing it is the best option for now. Another solution would be to set the filter as an attribute for the player, so when a new texture is created, it can be applied